src/librbd/AioImageRequestWQ.cc

```diff
@@ -121,7 +121,7 @@ void AioImageRequestWQ::aio_read(AioCompletion *c, uint64_t off, uint64_t len,
 
   if (m_image_ctx.non_blocking_aio || writes_blocked() || !writes_empty() ||
       lock_required) {
-    queue(new AioImageRead<>(m_image_ctx, c, {{off, len}}, buf, pbl, op_flags));
+    queue(new AioImageRead<>(m_image_ctx, c, {{off, len}}, buf, pbl, op_flags, trace_info));
   } else {
     c->start_op();
     AioImageRequest<>::aio_read(&m_image_ctx, c, {{off, len}}, buf, pbl,
```

Comment:
It was missing in this branch. Added here https://github.com/ceph/ceph/pull/11053/commits/adc72d6a1813cff0610d8398c783f65de7b0c8ce